packages/main/src/CheckBox.js

```diff
@@ -56,6 +56,26 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		* Determines whether the component is displayed as partially selected.
```

Comment:
Determines -> Defines
partially selected -> partially checked (2 places in the description)